src/ethereum_test_forks/forks/forks.py

```diff
@@ -543,6 +543,11 @@ def pre_allocation_blockchain(cls) -> Mapping:
         """
         return {}
 
+    @classmethod
+    def engine_get_blobs_version(cls, block_number: int = 0, timestamp: int = 0) -> Optional[int]:
+        """At genesis, blobs cannot be retrieved through the engine API."""
+        return None
+
```

Comment:
We can remove `engine_get_blobs_version` entirely from this PR as it is not really used and it's already part of #1644.